lib/models/channels_response.ex

```diff
@@ -0,0 +1,33 @@
+defmodule ExMicrosoftBot.Models.ChannelsResponse do
```

Comment:
If this is specific to Teams then can your make the module to be `ExMicrosoftBot.Models.Teams.ChannelsResponse` so that its very clear its specific to a certain app.